rasa/nlu/classifiers/embedding_intent_classifier.py

```diff
@@ -0,0 +1,167 @@
+        # the number of hidden layers is thus equal to the length of this list
+        HIDDEN_LAYERS_SIZES_TEXT: [],
+        # sizes of hidden layers before the embedding layer for intent labels
+        # the number of hidden layers is thus equal to the length of this list
+        HIDDEN_LAYERS_SIZES_LABEL: [],
+        # Whether to share the hidden layer weights between input words and labels
+        SHARE_HIDDEN_LAYERS: False,
+        # number of units in transformer
+        TRANSFORMER_SIZE: 256,
+        # number of transformer layers
+        NUM_TRANSFORMER_LAYERS: 2,
+        # number of attention heads in transformer
+        NUM_HEADS: 4,
+        # max sequence length if pos_encoding='emb'
+        MAX_SEQ_LENGTH: 256,
+        # training parameters
+        # initial and final batch sizes - batch size will be
+        # linearly increased for each epoch
+        BATCH_SIZES: [64, 256],
+        # how to create batches
+        BATCH_STRATEGY: "balanced",  # string 'sequence' or 'balanced'
+        # number of epochs
+        EPOCHS: 300,
+        # set random seed to any int to get reproducible results
+        RANDOM_SEED: None,
+        # optimizer
+        LEARNING_RATE: 0.001,
+        # embedding parameters
+        # default dense dimension used if no dense features are present
+        DENSE_DIM: {"text": 512, "label": 20},
+        # dimension size of embedding vectors
+        EMBED_DIM: 20,
+        # the type of the similarity
+        NUM_NEG: 20,
+        # flag if minimize only maximum similarity over incorrect actions
+        SIMILARITY_TYPE: "auto",  # string 'auto' or 'cosine' or 'inner'
+        # the type of the loss function
+        LOSS_TYPE: "softmax",  # string 'softmax' or 'margin'
+        # number of top intents to normalize scores for softmax loss_type
+        # set to 0 to turn off normalization
+        RANKING_LENGTH: 10,
+        # how similar the algorithm should try
+        # to make embedding vectors for correct labels
+        MU_POS: 0.8,  # should be 0.0 < ... < 1.0 for 'cosine'
+        # maximum negative similarity for incorrect labels
+        MU_NEG: -0.4,  # should be -1.0 < ... < 1.0 for 'cosine'
+        # flag: if true, only minimize the maximum similarity for incorrect labels
+        USE_MAX_SIM_NEG: True,
+        # scale loss inverse proportionally to confidence of correct prediction
+        SCALE_LOSS: True,
+        # regularization parameters
+        # the scale of L2 regularization
+        C2: 0.002,
+        # the scale of how critical the algorithm should be of minimizing the
+        # maximum similarity between embeddings of different labels
+        C_EMB: 0.8,
+        # dropout rate for rnn
+        DROPRATE: 0.2,
+        # use a unidirectional or bidirectional encoder
+        UNIDIRECTIONAL_ENCODER: True,
+        # if true apply dropout to sparse tensors
+        SPARSE_INPUT_DROPOUT: True,
```

Comment:
I think the old behavior would be false
```suggestion
        SPARSE_INPUT_DROPOUT: False,
```